apps/src/templates/manageStudents/ManageStudentsNameCell.jsx

```diff
@@ -0,0 +1,47 @@
+import React, {Component, PropTypes} from 'react';
+import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
+import {tableLayoutStyles} from "../tables/tableConstants";
+import i18n from "@cdo/locale";
+
+class ManageStudentNameCell extends Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    sectionId: PropTypes.number,
+    name: PropTypes.string.isRequired,
+    loginType: PropTypes.string.isRequired,
+    username: PropTypes.string,
+    isEditing: PropTypes.bool,
+  };
+
+  state = {
+    nameValue: this.props.name
+  };
+
+  onChangeName = (e) => {
+    this.setState({nameValue: e.target.value});
+  }
+
+  render() {
+    const {id, sectionId, name, loginType, username} = this.props;
+    const url = `/teacher-dashboard#/sections/${sectionId}/student/${id}`;
```

Comment:
You only use this in one place (line 32). I'd just do it inline there, though I'd also have 1 prop per line for that anchor tag.